ui/src/components/modals/DeleteModelPyFuncV2Modal.js

```diff
@@ -0,0 +1,116 @@
+  const [inactiveJob, setInactiveJob] = useState([])
+  const [deleteConfirmation, setDeleteConfirmation] = useState('')
+
+  const [{ isLoading, isLoaded }, deleteModel] = useMerlinApi(
+    `projects/${model.project_id}/models/${model.id}`,
+    { method: "DELETE", addToast: true, mock: mocks.noBody },
+    {},
+    false
+  );
+
+  const [jobs] = useMerlinApi(
+    `/projects/${model.project_id}/jobs?model_id=${model.id}`,
+    { mock: mocks.jobList },
+    []
+  );
+
+  const isActiveJob = function(status) {
+    return ["pending", "running"].includes(status)
+  }
+
+  useEffect(() => {
+    setActiveJob(jobs.data.filter(item => isActiveJob(item.status)))
+    setInactiveJob(jobs.data.filter(item => !isActiveJob(item.status)))
+  }, [jobs])
+
+  useEffect(() => {
+    if (isLoaded) {
+      closeModal();
+      callback();
+    }
+  }, [isLoaded, closeModal, callback]);
+
+  return (
+    <EuiOverlayMask>
+      <EuiConfirmModal
+        title="Delete Model"
+        onCancel={closeModal}
+        onConfirm={deleteModel}
+        cancelButtonText="Cancel"
+        confirmButtonText="Delete"
+        buttonColor="danger"
+        confirmButtonDisabled={deleteConfirmation !== model.name || activeJob.length > 0}>
+        {isLoading && (
+          <EuiProgress
+            size="xs"
+            color="accent"
+            className="euiProgress-beforePre"
+          />
+        )}
+        {jobs.isLoading ? (
+          <EuiProgress
+          size="xs"
+          color="accent"
+          className="euiProgress-beforePre"
+          />
+        ) : (
+          <div>
+            {activeJob.length > 0 ? (
+              <div>
+                You cannot delete this Model because there are <b> {activeJob.length} Active Prediction Jobs</b> using this model. 
+                <br/> <br/> If you still wish to delete this model, please <b>Terminate</b> Prediction Jobs that use this model or wait until the job completes.         
+              </div>
+              
+            ) : (
+              <div>
+                You are about to delete model <b>{model.name}</b>. This action cannot be undone. 
+                
+                <br/> <br/> To confirm, please type "<b>{model.name}</b>" in the box below
+                  <EuiFieldText     
+                    fullWidth            
+                    placeholder={model.name}
+                    value={deleteConfirmation}
+                    onChange={(e) => setDeleteConfirmation(e.target.value)}
+                    isInvalid={deleteConfirmation !== model.name} />  
+              </div>
+            )}
+            {activeJob.length === 0 && inactiveJob.length > 0 && (
+                <span>Deleting this Model will also delete {inactiveJob.length} <b>Inactive</b> Prediction Jobs using this model. <br/> <br/></span>
+            )}
+            {/* <JobsTableModelVersion jobs={activeJob.length > 0 ? activeJob : inactiveJob} isLoaded={jobs.isLoaded} error={jobs.error} /> */}
```

Comment:
Is the decision to NOT show the individual job info for delete models (as there can be too many)? If so, I agree with this. Let's remove the commented out lines though.